src/validation.cpp

```diff
@@ -1894,10 +1894,11 @@ void StopScriptCheckWorkerThreads()
 class WarningBitsConditionChecker : public AbstractThresholdConditionChecker
 {
 private:
-    int bit;
+    const ChainstateManager& m_chainman;
```

Comment:
Perhaps we can use a `const VersionBitsCache&` here?

Reply:
Could, but it should all get optimised away since `WarningBitsConditionChecker` is only briefly instantiated while it's being checked.

Reply:
Sure!